docs/start/install.mdx

```diff
@@ -8,6 +8,14 @@ description: Install and set up Qiskit and Qiskit Runtime on various operating s
 
 Whether you will work locally or in a cloud environment, the first step for all users is to install Qiskit. For those wanting to run on a real system, your next step is to choose one of two channels in order to access IBM Quantum systems: IBM Quantum Platform or IBM Cloud.
 
+<span id="qiskit-upgrade"></span>
+## Upgrade to Qiskit 1.0
```

Comment:
```suggestion
## Upgrade to Qiskit 1.0 requires deleting virtual environment
```

Reply:
@Eric-Arellano the blog doesn't mention explicitly deleting the current virtual environment. Is that implied when creating a new one? And do we need extra instructions on how to delete the current one?

Reply:
I'm not sure how best to do this, but I think we should make it clear somehow that this info is important for _existing_ users of qiskit. Brand new users can use the install page as normal. Maybe just the header should be changed, or else the first line could tell people to skip this section if they are a new user?